src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md

```diff
@@ -3,26 +3,28 @@ title: 'Requirements'
 excerpt: 'Requirements for using xk6-disruptor in your test scripts'
 ---
 
-The xk6-disruptor is a k6 extension. 
-To use it in a k6 test script, you need to bundle a k6 extension that includes the disruptor.
-Refer to the [Installation](/javascript-api/xk6-disruptor/get-started/installation) section  for instructions on how to get this custom build.
+At this moment,
+xk6-disruptor is a k6 extension **intended for systems running Kubernetes version 1.23 or higher**.
 
- xk6-disruptor needs to interact with the Kubernetes cluster on which the application under test is running.
- To do so, you must have the credentials to access the cluster in a [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) file.
- Ensure that this file is pointed to by the `KUBECONFIG` environment variable or that it is located at the default location, `$HOME/.kube/config`.
 
-<Blockquote mod="note">
+To use xk6-disruptor in a k6 test script, you need the following:
 
-xk6-disruptor requires Kubernetes version 1.23 or higher
+- **A bundled k6 extension that includes the disruptor.**
+Refer to the [Installation](/javascript-api/xk6-disruptor/get-started/installation) section  for instructions on how to get this custom build.
 
-</Blockquote>
+- **Credentials to access the cluster in a [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) file.**
+ Ensure that this file is pointed to by the `KUBECONFIG` environment variable or that it is located at the default location, `$HOME/.kube/config`.
 
+- **Access to the [grafana/xk6-disruptor-agent](https://github.com/grafana/xk6-disruptor/pkgs/container/xk6-disruptor-agent) image used by the [disruptor agent](/javascript-api/xk6-disruptor/explanations/how-xk6-disruptor-works/).**
 
-`xk6-disruptor` requires the [grafana/xk6-disruptor-agent](https://github.com/grafana/xk6-disruptor/pkgs/container/xk6-disruptor-agent) image for injecting the [disruptor agent](/javascript-api/xk6-disruptor/explanations/how-xk6-disruptor-works) into the disruption targets. Kubernetes clusters can be configured to restrict the download of images from public repositories. You need to ensure this image is available in the cluster where the application under test is running. Additionally, the xk6-disruptor-agent must run with network access privileges. Kubernetes clusters [can be configured to restrict the privileges of containers](https://kubernetes.io/docs/concepts/security/pod-security-admission/).
```

Comment:
I think you are removing relevant details here. Why is this image needed? I prefer the original phrasing.